payments/verifier.go

```diff
@@ -0,0 +1,41 @@
+package payments
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/utils/httpsignature"
+)
+
+// the list of payment verifiers, mapping to individuals in payments-ops
```

Comment:
Can you expand this comment a bit further? Are these keys that are managed by two members in Brave on the payment-ops team that we're just hardcoding in here for simplicity because we don't expect this list to be large.